wwahammy/houdini
# webhooks  to merge into add dispute  new

- Correct the name of a spec file and then fix the spec
- Simplify an #present? expectation
- Remove unused Coupon
- Named the slugged_event route and used it for a event-related social button share
- Use mailto link on email share button for events
- Add Disallow Anonymous Contributions to widgets
- Bump puma from 5.6.2 to 5.6.4
- Update Node 14 to latest
- Remove @types/moment-timezones as it's unneeded
- Update moment-timezone so it has the typings we need
- Correctly update to the proper version
- Add Supporter#calculated_first_name and #calculated_last_name
- Backport ActionController::Base#render
- Cleanup has_many factories
- Move recurring donation cancel into its own method
- Remove the persisted? check in RecurringDonation#cancel!
- Update to Ruby 2.5.9
- Add Supporter.undeleted_tag_masters, .email_lists, .active_email_lists
- Add way to query for recurring donations cancelled before a particular time
- Add PeriodicReportAdapter::CancelledRecurringDonationsReport
- Merge in Latest_supporter_level_goal
- Correct donation_base
- Bump nokogiri from 1.10.10 to 1.12.5
- Fix typo that prevented notes from being added to supporters on failed recurring donations
- Add cancelled_recurring_donations as available report type for periodic reports
- Add ability to pass in specific a custom Ruby version for testing with the CUSTOM_RUBY_VERSION env variable
- Create Ruby version testing matrix for Ruby 2.5, 2.6 and 2.7
- Set Ruby version to 2.6.10
- Update to a version of pg compatible with building on Ruby 2.6 with jemalloc
- Add monkeypatch for ActionController::TestResponse on Ruby 2.6
- Remove 2.5.9 from the build matrix
- Correct the README.md
- Swap byebug and pry for debug gem
- Update nokogiri
- Fix for change in how JSON sub-objects are returned by the server in Ruby 2.6
- Update Rack
- Removed unused credit-card-validator.js file
- Create custom mail for cancelled recurring donations report
- Add ActiveJob and create job for MailchimpSignupJob
- Add debugging for Mailchimp list models (#332)
- Fix for CVE-2020-5267
- Update rspec and friends to latest
